wiki/Mascots/en.md

```diff
@@ -64,36 +64,36 @@ Agent J was also featured in LuigiHann's skin, [Elite Beat osu! HD (1.0 Complete
 
 ## Community
 
 ### Aiko
 
 ![Aiko](img/Aiko.png "Aiko")
 
-Designed by [JMC](https://osu.ppy.sh/users/774010), Aiko was one of the participants in the osu!taiko mascot design contest. She is an energetic girl with a passion for osu!taiko, even if she isn't the best at it! Sporting a pair of stylish "Tabi" shoes and a retinue of pippidon accessories, the mascot of yore lives on in this spunky girl. She's fairly short at only 154cm, and was born on 1999-04-06.
+Designed by [JMC](https://osu.ppy.sh/users/774010), Aiko was one of the participants in the osu!taiko mascot design contest. She is an energetic girl with a passion for osu!taiko, even if she is not the best at it! Sporting a pair of stylish "Tabi" shoes and a retinue of pippidon accessories, the mascot of yore lives on in this spunky girl. She is fairly short at only 154 centimetres, and was born on 1999-04-06.
 
 ### Alisa
 
 ![Alisa](img/Alisa.png "Alisa")
 
-Designed by [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa was one of the participants in the osu!taiko mascot design contest. She has been playing osu!taiko ever since her early childhood. Fond of playing songs for others, she is very musically oriented, and when she's not sleeping or eating, she's playing osu!taiko, or playing retro video games for kicks!
+Designed by [\[ Glitch \]](https://osu.ppy.sh/users/3781400), Alisa was one of the participants in the osu!taiko mascot design contest. She has been playing osu!taiko ever since her early childhood. Fond of playing songs for others, she is very musically oriented, and when she is not sleeping or eating, she is playing osu!taiko, or playing retro video games for kicks!
 
-### Tama
+### Chirou
 
-![Tama](img/Tama.png "Tama")
+![Chirou](img/Chirou.png "Chirou")
 
-Designed by [crystalsuicine](https://osu.ppy.sh/users/9974), Tama was one of the younger participants in the osu!taiko mascot design contest at only age 15 (or is she?), Tama has a huge passion for taiko drumming. And thunderstorms. And especially festivals, where she snatches up any takoyaki she can find. Always up for the challenge, Tama hides a particularly mysterious past behind her youthful demeanor.
+Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the participants in the osu!taiko mascot design contest. She is very strict and demanding, a perfectionist, and does not like making mistakes—especially in her beats in osu!taiko. But past her strong outlook, if you get to her soft spot, she can be fluffy and cute. She is 14 years old, born on October 25th with blood type AB. She is also 4 feet 11 inches and weighs 100 pounds. Her hobbies include practicing drumming, keeping warm in her cape, and collecting gems or rocks. Chirou has made some appearances in various user fanart submissions.
```

Comment:
I think it should be metric (for consistency).
`4 feet 11 inches and weighs 100 pounds` -> `about 149.86 centimetres and weighs about 45.36 kilograms` (taken from previous commits)
```suggestion
Designed by [pyun](https://osu.ppy.sh/users/981534), Chirou was one of the participants in the osu!taiko mascot design contest. She is very strict and demanding, a perfectionist, and does not like making mistakes—especially in her beats in osu!taiko. But past her strong outlook, if you get to her soft spot, she can be fluffy and cute. She is 14 years old, born on October 25th with blood type AB. She is also about 149.86 centimetres and weighs about 45.36 kilograms. Her hobbies include practicing drumming, keeping warm in her cape, and collecting gems or rocks. Chirou has made some appearances in various user fanart submissions.
```

Reply:
it's a character profile, typically these just include what the artist intended

not that units change the meaning but saying "149.86 cm" makes it look like that was intentional when it's not 🤔

Reply:
probably round to whole number. otherwise, it's fine.